src/components/primitives/Text/Text.js

```diff
@@ -1,38 +1,47 @@
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
-import { space, variant } from 'styled-system';
+import { color, typography, space, variant } from 'styled-system';
+import { css } from '@styled-system/css';
 
 /**
  * A component to render all text in the app.
  */
 const Text = styled.p(
-	{
-		margin: 0,
-		listStyleType: 'none',
-		textAlign: props => props.align,
-	},
+	// Default styles
+	// css function provides access to our design tokens
+	css({
+		m: 0,
+		fontWeight: 'normal',
+		fontFamily: 'body',
+		fontSize: 'md',
+		lineHeight: 'base',
+		color: 'base',
+	}),
+	// APIs
+	color,
+	typography,
 	space,
 	variant({
 		variants: {
 			base: {
+				fontSize: 'md', // You can use design tokens in variants!
 				fontWeight: 'normal',
 				lineHeight: 'base',
 				fontFamily: 'body',
-				fontSize: 'md',
-				color: 'primary',
+				color: 'text',
```

Comment:
Looks like we don't have `text` color?

Reply:
Good catch. I wanted to rename `base` to `text` to be conformed with styled-system theme spec but forgot. Will update it.